src/core/instance/state.js

```diff
@@ -147,7 +147,16 @@ function initComputed (vm: Component, computed: Object) {
 
   for (const key in computed) {
     const userDef = computed[key]
-    const getter = typeof userDef === 'function' ? userDef : userDef.get
+    let getter = typeof userDef === 'function' ? userDef : userDef.get
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
I don't think so. Former is better as define plugin would replace `process.env.NODE_ENV` with `'production'` and it can be stripped off.